### PR TITLE
Gameoverscreen added, fixes #66

### LINKED ISSUE #66: Add game over screen
As a user I want to be able to see my score at the game over screen

### PR DESCRIPTION
**Pull request by Zerochaosz**

### CODE REVIEW

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    public GameOverScreen() {
+        try {
+            backgroundImage = ImageIO.read(Level.class.getResource("/sprites/gameover.png"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        //These are from right to left.
+        Places = new ArrayList<>(9);
+        NumberToken nine = new NumberToken(524,300);
```
Comment (Wolgo) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: Same as in WinScreen, (which i guess you helped Tim with), why not:
`Places.add( new NumberToken(492,300))` etc.
and 
```
for (NumberToken token : Places()) {
            output.objectList.add(token);
        }
```

**Comment by Wolgo** [comment]
Aside from that it looks good.

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java (hunk 1 above)
Comment (TMRert) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: indeed, or use this to create the arrays
```java
int numberPosX = 524;
int numberPosY = 300
int numberOfTokens = 9;
for(int i = 0; i<numberOfTokens; i++) {
    NumberToken token = new NumberToken(numberPosX, numberPosY);
    Places.add(token);
    objectList.add(token);
    numberposX -= 32;
}

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import java.util.ArrayList;
+
+/**
+ * Class that represents the game over screen.
+ * @author Jaap-Jan
+ * @date 21-9-2016.
+ */
+@Slf4j
+public class GameOverScreen extends Screen {
+    private static ArrayList<NumberToken> Places;
```
Comment (TMRert) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: Please note that variables are denoted in camelCase, not in PascalCase

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java (hunk 1 above)
Comment (TMRert) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: of course, to resolve checkstyle magic number errors, numberPosX, numberPosY, numberOfTokens and the -32 should be declared in top of the file as private static final integers

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java (hunk 2 above)
Comment (Wolgo) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: I think that's a copy paste error though, which uh, is totally my fault.

Comment (Fastjur) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: Let's fix this before merging

File: src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java (hunk 1 above)
Comment (Fastjur) [inline src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java]: Let's fix this before merging

**Comment by Zerochaosz** [comment]
This should fix all of the problems

**Comment by Wolgo** [comment]
That has fixed the problems. @Fastjur could your remove your change request.

**Pull request merged**